_targets.R

```diff
@@ -0,0 +1,34 @@
+library(targets)
+source('code.R')
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch_data(project_output_dir = "1_fetch/out/model_RMSEs.csv"),
```

Comment:
Nice work adapting this code to work with your functions. You're passing the full filename of the output file here, which is exactly in line with best practices for pipelines. However, you're still naming the argument `project_output_dir`, even though you're now passing in the filename, rather than the name of the directory.  Could you edit the argument name to reflect what is being passed?